kimchi/src/circuits/polynomials/range_check/gate.rs

```diff
@@ -1,1400 +0,0 @@
-//! Range check gate
```

Comment:
it stil shows this file as deleted, and the gadget file as created from scratch. I think you need to do a `git mv`

Reply:
I think it's because you do the `git mv` too late in your commits. You need to have the `git mv` in the first commit

Reply:
Never mind, it is named now again as gate.rs, so it will show the diff

Reply:
btw if I were you, to fix the issue I would:

1. create a clean branch where I `git mv gate.rs gadget.rs`
1. then merge this branch into the clean branch (or better, squash all these commits, and then merge into the clean branch)